spark-operator/src/main/java/org/apache/spark/k8s/operator/metrics/MetricsSystemFactory.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.spark.k8s.operator.metrics;
+
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+import org.apache.commons.lang3.StringUtils;
+
+import org.apache.spark.k8s.operator.config.SparkOperatorConfManager;
+
+/** Factory for MetricsSystem */
+public class MetricsSystemFactory {
+  public static final String METRIC_PREFIX = "spark.metrics.conf.operator.";
+  public static final String SINK = "sink.";
+  public static final String CLASS = "class";
+
+  public static MetricsSystem createMetricsSystem() {
+    Properties properties =
+        parseMetricsProperties(SparkOperatorConfManager.INSTANCE.getMetricsProperties());
+    return new MetricsSystem(properties);
+  }
+
+  public static MetricsSystem createMetricsSystem(Properties properties) {
+    return new MetricsSystem(properties);
+  }
+
+  private static Properties parseMetricsProperties(Properties userProperties) {
+    Properties properties = new Properties();
+    Enumeration<?> valueEnumeration = userProperties.propertyNames();
+    while (valueEnumeration.hasMoreElements()) {
+      String key = (String) valueEnumeration.nextElement();
+      if (key.startsWith(METRIC_PREFIX)) {
+        properties.put(key.substring(METRIC_PREFIX.length()), userProperties.getProperty(key));
+      }
+    }
+    return properties;
+  }
+
+  public static Map<String, MetricsSystem.SinkProps> parseSinkProperties(
+      Properties metricsProperties) {
+    Map<String, MetricsSystem.SinkProps> propertiesMap = new HashMap<>();
+    // e.g: "sink.graphite.class"="org.apache.spark.metrics.sink.GraphiteSink"
```

Comment:
Can we use a simpler example like `org.apache.spark.metrics.sink.ConsoleSink` instead of `GraphiteSink`?